.jenkins/pytorch/test.sh

```diff
@@ -22,7 +22,7 @@ if [[ "$BUILD_ENVIRONMENT" == *coverage* ]]; then
   export PYTORCH_COLLECT_COVERAGE=1
 fi
 
-if [[ "$BUILD_ENVIRONMENT" == *cuda11.1* ]]; then
+if [[ "$BUILD_ENVIRONMENT" == *cuda11* ]]; then
```

Comment:
any particular reason why using `cuda11` here and the window side uses `cuda11.2`?

Reply:
No particular reason--I was actually going to change it to just 11 in linux in the linux PR :p